src/embedCreators/TriggersEmbedCreator.ts

```diff
@@ -0,0 +1,24 @@
+﻿import { EventData } from '../models/internal-models';
+import BaseEmbedCreator from './BaseEmbedCreator.js';
+import HeroesTriggers from '../triggers/heroes-trigger-templates.js';
+
+export class TriggersEmbedCreator extends BaseEmbedCreator {
+    constructor(languageLocation: string, data: EventData) {
+        super(languageLocation, data);
+    }
+    
+    addTriggersWithSlash(slash: string) {
+        this.setNameAndDescription(slash, "translatedArtifact.translatedDescription");
+        let fieldString = '';
+        for (const value of Object.values(HeroesTriggers)) {
```

Comment:
Can this be made as an Util so that this could be used for different content in addition to artifacts